Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 ruby '3.0.1'
 
 # Backend
+gem 'activerecord-import'
```

Comment:
Always add a comment to each gem to provide information on why the gem was added (see the either gems for example)